### PR TITLE
Completed html-forms

### PR DESCRIPTION
**Pull request by K-Bowler**
html-forms

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>HTML-Forms</title>
+</head>
+<body>
+  <h1>Coding Form</h1>
+  <form>
+    <label for="email">Email</label> <br>
+    <input id="email" type="text" name="email"> <br>
```
Comment (BrettAlbright) [inline html-forms/index.html]: you want to avoid using `br` tags, div tags are the proper way to break things onto multiple lines

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

File: html-forms/index.html (hunk 1 above)
Comment (K-Bowler) [reply html-forms/index.html]: image

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**